app/templates/viewAssignment.html

```diff
@@ -9,4 +9,13 @@ <h1 class="page-title">{{assignment.name}}</h1>
     </div>
 </div>
 
+<div ng-if="assignment.question_type == 'poll'" class="poll-results">
+    <br><br><br>
+    <p>{{assignment.questions[0].answer0}} = {{assignment.answer0_count}}</p>
+    <p>{{assignment.questions[0].answer1}} = {{assignment.answer1_count}}</p>
+    <p>{{assignment.questions[0].answer2}} = {{assignment.answer2_count}}</p>
+    <p>{{assignment.questions[0].answer3}} = {{assignment.answer3_count}}</p>
+    <p>{{assignment.questions[0].answer4}} = {{assignment.answer4_count}}</p>
+</div>
+
```

Comment:
@Nolski this is where the grossness lives ... if we could wrap this in something JS'y and sexy that would be amazeballs.